icommons_ext_tools/settings/base.py

```diff
@@ -55,13 +63,20 @@
 # environment settings files to point to the environment-specific log directory.
 # Here in the base settings it's set explicitly to None so it will throw an
 # Exception unless overridden in individual environment settings
-EMAIL_FILE_PATH = None
+EMAIL_FILE_PATH = LOG_ROOT
+
 # Use smtp.EmailBackend with EMAIL_HOST and EMAIL_USE_TLS
 # to send actual mail via SMTP
+# Note that if DEBUG = True (because these are the integration test settings),
```

Comment:
The specific text about 'integration test settings' is misleading now that this comment is in base.py.